### DISCUSSION TITLE
What is the meaning of multiple connections in a data source

### DISCUSSION THREAD

**Discussion by nnikopoulos** [ariacom/Seal-Report#126] [General]
I am really struggle to find any meaning why a datasource can have multiple database connections! The logical would be a 1-1 between datasource and connection. The only reason I could think of is to be able to connect data from different databases, but that doesn't make sense in this case since only one connection can be active to a data source.

**Comment by ariacom** [comment]
It is (VERY) useful when you have several environments : a database for DEV, a database for TEST a database for PROD...
or a database freeze at 31-12 of the year...

**Reply by nnikopoulos** [reply]
OK. I see. Otherwise, you would have to create separate data sources for each of these cases, while in essence they are intended for the same concept, and if you did this for every new concept then chaos would prevail. Thanks.
